spec/fortnox/api/repositories/invoice_spec.rb

```diff
@@ -1,10 +1,20 @@
 require 'spec_helper'
-require 'fortnox/api/repositories/context'
+require 'fortnox/api/repositories/environment_context'
 require 'fortnox/api/repositories/invoice'
-require 'fortnox/api/models/invoice'
+require 'fortnox/api/repositories/examples/all_examples'
+require 'fortnox/api/repositories/examples/find_examples'
+require 'fortnox/api/repositories/examples/save_examples'
 
 describe Fortnox::API::Repository::Invoice do
-  include_context 'repository context'
+  include_context 'environment context'
 
-  it_behaves_like 'repositories', Fortnox::API::Model::Invoice
+  include_examples '.all'
+
+  include_examples '.find'
+
+  include_examples '.save', :comments do
+    let( :value ){ 'An invoice comment' }
+    let( :updated_value ){ 'An updated invoice comment' }
+    let( :model_hash ){ { comments: value, customer_number: 1 } }
```

Comment:
Why are we still creating this outside the example group? Can't you just include this in the example group as well? If you need to send the id in (which I guess you don't right now) you can always add it as another prop you send in.

I'm also thinking that the value isn't that important? The specific value I mean. As long as the example group throws an exception if the type of the value isn't a string (or doesn't respond to `.to_s`) we should be safe. Most properties on models are strings so finding a property for the test for any model that can take a random sting shouldn't be a problem. That way you can just use `"A value"` and `"A changed value"` statically in the example group and this would all just become:

```
include_examples '.save', :comments
```
Like the rest of the include_example lines ...

Arguments against this are welcome however. I'm not sure it's a good idea, but it would make the tests consistent and nice looking :)